src/Fable.Cli/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+
+[JS/TS] Fixed `DateTime.Add` for `DateTimeKind.Unspecified` (by @ncave)
+[Rust] Fixed deprecated `NaiveDateTime` usage in `DateTime` (by @ncave)
```

Comment:
Thank you for looking into the Date issue.

These lines are missing `* ` at the beginning of them

Reply:
Thanks, fixed.